.github/workflows/wheel-builder.yml

```diff
@@ -55,14 +55,17 @@ jobs:
           path: cryptography-wheelhouse/
 
   macos:
-    runs-on: macos-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
There's no need to do change our wheel builder, the wheels we build already work with any macOS version 10.10 and greater.

Reply:
I should probably have opened an issue first, but the prebuilt-wheel does not work on mac 11, it tries to re-compile

Reply:
Trying upgrading to a newer pip. It sounds like you're hitting https://github.com/pypa/pip/issues/9138

Reply:
Fair enough, I'll remove the duplicated release job then!

Reply:
Actually the fixed is not released yet, it should come around pretty soon though